src/data/products.json

```diff
@@ -0,0 +1,13 @@
+[
```

Comment:
added a test data

Reply:
🟡 this will conflict with TES-31, as I think the data should have a bit more detailed structure (each color version should have its own stock amount to be more realistic, and the "in stock" value is not needed in the data, but can be set as a boolean based on the stock amount), but I'll take care of that in the respective task. Good that the Image component works :)